components/vf-summary/vf-summary.scss

```diff
@@ -38,6 +62,13 @@
   color: set-color($vf-summary__date-color--text);
 }
 
+.vf-summary__source {
+  @include set-type($vf-summary__date-typography);
+
+  color: set-color($vf-summary__date-color--text);
+  font-style: italic;
```

Comment:
Feels weird using italic as we don't anywhere else, but I suppose that's a comment to direct to the design pattern/Mark.

Reply:
Which I asked already ;)

Reply:
checking this in Firefox using the fonts panel in the dev tools -- this is using the correct IBM Plex Sans Italic font file